sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py

```diff
@@ -847,10 +852,18 @@ async def download_recording(
         :rtype: AsyncIterable[bytes]
         :raises ~azure.core.exceptions.HttpResponseError:
         """
-        stream = await self._downloader.download_streaming(
-            source_location=recording_url, offset=offset, length=length, **kwargs
+        response = await self._downloader.download_streaming(
```

Comment:
`download_streaming` will raise errors for any of these values being  None, so is passing in 0 proper here or it should raise an error ?

https://github.com/Azure/azure-sdk-for-python/blob/5a7eca56c1756de57d3a743955ae44244cb6ff23/sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_content_downloader.py#L35